extensions/grpc/deployment/src/main/resources/dev-templates/service.html

```diff
@@ -99,7 +99,8 @@
 
               connection.responseText = data.body + connection.responseText;
               responseElement.value = connection.responseText;
-              responseElement.style.height = Math.min(responseElement.scrollHeight + 5, 500) + "px"
+              responseElement.style.height = "1px";
```

Comment:
Why having this assignation? Isn't it overridden on the next line.

Reply:
that's the trick :)

This code uses `responseElement.scrollHeight` to determine how big a textarea needs to be to fit the text without scrolling.

Setting `responseElement.style.height` impacts the value of `responseElement.scrollHeight`. So when this code is invoked twice, if we don't reset to 1px (or basically, anything else), the `responseElement.scrollHeight` will show the larger value. And on each invocation the size of the textarea will grow.